streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -69,6 +73,7 @@
 import org.apache.kafka.streams.state.internals.QueryableStoreProvider;
 import org.apache.kafka.streams.state.internals.RocksDBGenericOptionsToDbOptionsColumnFamilyOptionsAdapter;
 import org.apache.kafka.streams.state.internals.StreamThreadStateStoreProvider;
+import org.apache.kafka.common.errors.TimeoutException;
```

Comment:
nit: put this import above with the others (IDE often misplaces these since we follow a weird import ordering in places)